index.d.ts

```diff
@@ -8,9 +8,15 @@ import {
 // NOTE: there is a lot of repetition here :(
 // Repetition will be less when ember-statecharts _is_ TS?
 
-type WithConfigParams<Context, Schema, Events extends EventObject> = Parameters<StateMachine<Context, Schema, Events>['withConfig']>;
-type WithContextParams<Context, Schema, Events extends EventObject> = Parameters< StateMachine<Context, Schema, Events>['withContext']>;
-type Send<Context, Schema, Events extends EventObject> = Interpreter<Context, Schema, Events>['send'];
+
+// Private Types
+// For building the Public Types?
+// C = Context
+// S = Schema
+// E = Events
+type WithConfigParams<C, S, E extends EventObject> = Parameters<StateMachine<C, S, E>['withConfig']>;
```

Comment:
I had to debug some types for a bit, and decided to abbreviate these. Nothing has changed though.